contracts/PerunArt.sol

```diff
@@ -5,23 +5,37 @@ pragma solidity ^0.8.0;
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
 
 contract PerunArt is ERC721 {
-    // The single allowed token minter
-    address public immutable minter;
+    address public immutable owner;
+    mapping(address => bool) minters;
```

Comment:
Is this really something which is only useful for tests? If not enlighten me with a use-case :D

Otherwise it feels weird to have it in production code. Currently, I am not really aware of how I would extend `PerunArt` minimally s.t. it is convenient for test usage, though.

Reply:
What do you mean? `PerunArt` itself will never be deployed to main net, it is only there to serve as an example contract of an `ERC721Mintable`. This mechanism was the easiest way to set minters (i.e., TokenHolders) by the operator (=owner) after deploying the Erdstall contracts (again and again) without the need to redeploy the `PerunArt` contract every time, like we also do for the `PerunToken` ERC20.

Reply:
Nevermind, I forgot about that and was a few too many steps in a potential future within another universe 🤡